doc/programming_guide/ffd/deploying/configuration.rst

```diff
@@ -102,31 +102,30 @@ The |I2C| master is used when the FFD example asynchronously sends intent messag
   The |I2C| interface cannot operate as both master and slave simultaneously. The FFD example design uses the |I2C| master interface to configure the DAC at device initialisation.
   However, if the host reads intent messages from the FFD example using the |I2C| slave interface, the |I2C| master interface will be disabled after the DAC configuration is complete.
 
-The |I2C| master and slave can be enabled or disabled by setting the ``appconfI2C_MASTER_ENABLED`` and ``appconfI2C_SLAVE_ENABLED`` configuration variables.
-
 To send the intent ID via the |I2C| master interface when a command is detected, set the following variables:
 
-  - ``appconfINTENT_I2C_OUTPUT_ENABLED`` to 1.
-  - ``appconfI2C_MASTER_ENABLED`` to 1.
-  - ``appconfINTENT_I2C_OUTPUT_DEVICE_ADDR`` to the desired address used by the |I2C| slave device.
-  - ``appconfI2C_SLAVE_ENABLED`` to 0.
+  - ``appconfINTENT_I2C_MASTER_OUTPUT_ENABLED`` to 1.
+  - ``appconfINTENT_I2C_MASTER_DEVICE_ADDR`` to the desired address used by the |I2C| slave device.
+  - ``appconfINTENT_I2C_SLAVE_POLLED_ENABLED`` to 0, this will disable the |I2C| slave interface.
 
 To configure the FFD example so that the host can poll for the intent via the |I2C| slave interface, set the following variables:
 
-  - ``appconfI2C_SLAVE_ENABLED`` to 1.
+  - ``appconfINTENT_I2C_SLAVE_POLLED_ENABLED`` to 1.
   - ``appconfI2C_SLAVE_DEVICE_ADDR`` to the desired address used by the |I2C| master device.
   - ``appconfINTENT_I2C_REG_ADDRESS`` to the desired register read by the |I2C| master device.
-  - ``appconfINTENT_I2C_OUTPUT_ENABLED`` to 0, this will disable the |I2C| master interface.
+  - ``appconfINTENT_I2C_MASTER_OUTPUT_ENABLED`` to 0, this will disable the |I2C| master interface after initialization.
 
 The handling of the |I2C| slave registers is done in the ``examples\ffd\src\i2c_reg_handling.c`` file. The variable ``appconfINTENT_I2C_REG_ADDRESS`` is used in the callback function ``read_device_reg()``.
 
 Configuring the |I2S| interface
 -------------------------------
 
-The |I2S| interface is used to receive the audio data from the host. The |I2S| interface can be configured as either a master or a slave.
+The |I2S| interface is used to send the intent audio to the DAC, and/or to receive the audio samples from the host. The |I2S| interface can be configured as either a master or a slave.
```

Comment:
I don't clearly understand the meaning of `intent audio`.

Reply:
I added more details in https://github.com/lucianomartin/sln_voice/pull/1/commits/e30e4603aca7a98dcbc5ec7757ef893a8840ee83.

I didn't manage to test the audio files, so I am not sure how they work. Let me know if I should create a story to understand them better.